programs/jit-proxy/src/lib.rs

```diff
@@ -70,13 +70,21 @@ pub mod jit_proxy {
         match maker_direction {
             PositionDirection::Long => {
                 if taker_price > maker_worst_price {
-                    msg!("taker price {} > worst bid {}", taker_price, maker_worst_price);
+                    msg!(
+                        "taker price {} > worst bid {}",
+                        taker_price,
+                        maker_worst_price
+                    );
                     return Err(ErrorCode::BidNotCrossed.into());
                 }
             }
             PositionDirection::Short => {
                 if taker_price < maker_worst_price {
-                    msg!("taker price {} < worst ask {}", taker_price, maker_worst_price);
+                    msg!(
+                        "taker price {} < worst ask {}",
+                        taker_price,
+                        maker_worst_price
+                    );
```

Comment:
can you revert this? just dont want to mix cross-code formatting with PR as much as possible

Reply:
unfortunately not bc of the pre commit hooks which i'm trying not to change